partner_statement/wizard/statement_common.py

```diff
@@ -14,7 +14,7 @@ class StatementCommon(models.AbstractModel):
     def _get_company(self):
         return (
             self.env["res.company"].browse(self.env.context.get("force_company"))
```

Comment:
@RujiaL I don't think this is correct in 14.0 anymore as force_company is deprecated. Probably just self.env.company as a default is fine and this function no longer needed.